site/content/contribute/mobile/push-notifications/migrate-gcm-fcm.md

```diff
@@ -9,7 +9,7 @@ As of April 10, 2018, Google has deprecated the [Google Cloud Messaging (GCM) se
 
 The [Mattermost Push Notification Service](https://github.com/mattermost/mattermost-push-proxy/releases) version 5.8+ uses FCM to send Push notifications for Android.
 
-1. Upgrade your Mattermost Push Notification service -> add link to upgrade section  to 5.8 or above.
+1. Upgrade your Mattermost Push Notification service to [version 5.8 or above](https://github.com/mattermost/mattermost-push-proxy/releases)
```

Comment:
Can we make "Upgrade your Mattermost Push Notification service" link to this heading in this doc:

![image](https://user-images.githubusercontent.com/14333569/54776770-2c0ba500-4be7-11e9-9087-a60369b9148a.png)?